data/strings/strings.txt

```diff
@@ -26763,3 +26781,8 @@
     comment = A number of bookmarks and a number of tracks, separated by comma, like: 1 bookmark, 5 tracks
     tags = android
     en = %@, %@
+
+  [app_tip_00]
+    comment = Tip #00
+    tags = android
+    en = Thank you for using our community-built maps!
```

Comment:
```
    ar = شكرًا لك على استخدام الخرائط التي أنشأها مجتمعنا!
    be = Дзякуй за выкарыстанне нашых карт, створаных супольнасцю!
    bg = Благодарим ви, че използвате нашите карти, създадени от общността!
    ca = Gràcies per utilitzar els nostres mapes creats per la comunitat!
    cs = Děkujeme, že používáte naše komunitní mapy!
    da = Tak, fordi du bruger vores community-byggede kort!
    de = Danke, dass du unsere von der gemeinschaft erstellten karten benutzt!
    el = Σας ευχαριστούμε που χρησιμοποιείτε τους χάρτες μας που δημιουργήθηκαν από την κοινότητα!
    es = ¡gracias por utilizar nuestros mapas creados por la comunidad!
    et = Aitäh, et kasutate meie kogukonna koostatud kaarte!
    eu = Eskerrik asko gure komunitateak eraikitako mapak erabiltzeagatik!
    fa = از شما برای استفاده از نقشه های ساخته شده توسط جامعه سپاسگزاریم!
    fi = Kiitos, että käytät yhteisön laatimia karttojamme!
    fr = Merci d'utiliser nos cartes créées par la communauté !
    he = תודה על השימוש במפות שנבנו בקהילה שלנו!
    hi = हमारे समुदाय-निर्मित मानचित्रों का उपयोग करने के लिए धन्यवाद!
    hu = Köszönjük, hogy használja a közösség által készített térképeinket!
    id = Terima kasih telah menggunakan peta buatan komunitas kami!
    it = Grazie per aver utilizzato le nostre mappe costruite dalla comunità!
    ja = 私たちのコミュニティが作った地図をご利用いただき、ありがとうございます！
    ko = 커뮤니티에서 제작한 지도를 이용해 주셔서 감사합니다!
    mr = आमचे समुदाय-निर्मित नकाशे वापरल्याबद्दल धन्यवाद!
    nb = Takk for at du bruker kartene vi har laget i fellesskap!
    nl = Bedankt voor het gebruik van onze door de community gemaakte kaarten!
    pl = Dziękujemy za korzystanie z naszych map stworzonych przez społeczność!
    pt = Obrigado por utilizares os nossos mapas construídos pela comunidade!
    pt-BR = Obrigado por você usar nossos mapas criados pela comunidade!
    ro = Vă mulțumim că utilizați hărțile create de comunitatea noastră!
    ru = Спасибо, что пользуетесь нашими картами, созданными сообществом!
    sk = Ďakujeme, že používate naše mapy vytvorené komunitou!
    sv = Tack för att du använder våra gemenskapsbyggda kartor!
    sw = Asante kwa kutumia ramani zetu zilizojengwa na jumuiya!
    th = ขอขอบคุณที่ใช้แผนที่ที่สร้างโดยชุมชนของเรา!
    tr = Topluluk tarafından oluşturulan haritalarımızı kullandığınız için teşekkür ederiz!
    uk = Дякуємо, що користуєтесь нашими мапами, створеними спільнотою!
    vi = Cảm ơn bạn đã sử dụng bản đồ do cộng đồng xây dựng của chúng tôi!
    zh-Hans = 感谢您使用我们的社区地图！
    zh-Hant = 感謝您使用我們社區構建的地圖！
```

Reply:
Small correction for german:
`de = Danke, dass du unsere von der Gemeinschaft erstellten Karten benutzt!`

Reply:
Thanks @biodranik, @Mr-Mime for the translations...

I will proceed with the translation of the rest of tips in [PR 5974](https://github.com/organicmaps/organicmaps/pull/5974).